src/app/static/src/app/components/plots/bar/Barchart.vue

```diff
@@ -50,7 +50,7 @@ export default defineComponent({
         });
 
         const chartDataGetter = store.getters["plotSelections/barchartData"];
-        const chartData = ref<BarChartData>({datasets:[], labels: [], maxValuePlusError: 0});
+        const chartData = ref<BarChartData<number[]>>({datasets:[], labels: [], maxValuePlusError: 0});
```

Comment:
I have put in a generic that lets us define the `data` type of the `datasets` key in `ChartData` type that `chart-js` exposes and we always give it a `number[]` from console logging the barchart data